### DISCUSSION TITLE
Prompt for unRAID server password

### DISCUSSION THREAD

**Discussion by pintapass** [drakkan/sftpgo#1537] [Q&A]
I'm getting to grips with my first-ever unRAID server and was going to use STFPgo (as a Docker app) to provide SFTP and WebDAV access to select folders in my unRAID array (well, ZFS pool, but kinda the same thing).

However, when I attempt to connect via SFTP (using the Forklift or Transmit clients on macOS) as the STFPgo admin or a user with the /unraid mount point, which should put me at the top level of all my data, I'm being prompted for a server password. I can't determine what this password is. It's not my unRAID root password, and it's not the SFTPgo admin password.

Am I missing something or misunderstanding how this should work?

**Comment by drakkan** [comment]
Hello,

I don't know anything about unRAID so I can't help you. unRAID is a paid product, you should ask them for support.
It is very easy to allow a software to be installed on a paid product and redirect your users upstream for support, which is why we are considering contacting unRAID, and other similar paid products, which include SFTPGo in their software catalog and ask to remove it

**Reply by pintapass** [reply]
No one from unRAID sent or directed me here. This was my first port of call, as I thought the issue I encountered might be common with Docker variants of this app. I'll take your advice and look to those communities for help instead.

**Comment by drakkan** [comment]
> No one from unRAID sent or directed me here. This was my first port of call, as I thought the issue I encountered might be common with Docker variants of this app. I'll take your advice and look to those communities for help instead.

If you search for SFTPGo in unRAID apps, the help link redirect here

https://unraid.net/community/apps?q=sftpgo

it should redirect to their community instead. The problem you are having may be specific to unRAID and since it is a proprietary product we cannot purchase it just to see what the problem is. Thanks for understanding
